tests/staticfiles_tests/test_storage.py

```diff
@@ -77,6 +77,15 @@ def test_path_ignored_completely(self):
             self.assertIn(b"url()", content)
         self.assertPostCondition()
 
+    @override_settings(STATIC_URL="/")
```

Comment:
Unfortunately this is not a proper regression test as `CollectionTestCase` overrides `STATIC_URL` again to the `static/` so inside `render_template()` it's no longer `/`.

Reply:
Ah, you're right! I'll work on a fix. I should have run the test to make sure it failed without my fix. Sorry about that!

Reply:
I moved the test to a new class in `3592c71`. Let me know what you think!